example/cumulus-tf/discover_and_queue_pdrs_workflow.tf

```diff
@@ -17,3 +17,22 @@ module "discover_and_queue_pdrs_workflow" {
     }
   )
 }
+module "discover_and_queue_pdrs_unique_workflow" {
```

Comment:
Very minor: We have other variants on discover_and_queue_pdrs defined separate terraform files. Should we do that here instead of lumping the new module in with the top-level terraform?

Doesn't really matter. Just organization.

Reply:
Hmm, I think org matters.   I'd argue for putting related setups in the same file, but you're right that that isn't the current pattern, and there's no reason not to follow it.

Reply:
58dfc83e1b6